actions/src/issues/checkIssueLabels.js

```diff
@@ -50,12 +50,14 @@ const checkLabels = async () => {
 const handleGoodFirstIssueLabel = async (octokit, user) => {
   const issueNumber = context.payload.issue.number;
   // Comment on the issue and ping the onboarding team lead.
+  var commentBody = (
+    'Hi @' + user + ', only certain users are allowed to add good ' +
+    'first issue labels. Looping in @oppia/oppia-good-first-issue-labelers ' + 
+    'to add the label.'
```

Comment:
Maybe add "Thanks!" at the end so the developer doesn't feel they did something wrong.